CONTRIBUTING.md

```diff
@@ -0,0 +1,133 @@
+  /fabric
+  /databricks
+    /<name-of-singletech-sample-1>
+    /<name-of-singletech-sample-2>
+  ...
+/utilities
+  /fabric
+    /<name-of-utility-1>
+    /<name-of-utility-2>
+  /databricks
+```
+
+**Fifth**, Create a Github issue using the "New Sample Setup" template and fill in the information required. This issue template also contains a detailed checklist to guide you in your sample development.
+
+### Getting Started
+
+Once you have done all of the above, see this [sample structure](/docs/sample_project_structure/README.md) to get started. Use that structure as a starting point.
+
+#### Your sample's README
+
+This is arguably the *most important* part of your sample. In general, it is recommended to put most of the content in a single README to avoid content being "lost" and skipped over through deep linking.
+
+Structure of the README:
+
+1. **Table of Contents** - Use [this VSCode extension](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) to generate TOC.
+1. **Solution Overview** - Provide the background of the solution (i.e. What problem are you trying to solve?), solution architecture and key technologies used.
+1. **Key Learnings** - These are the key takeaways you want to impart to the engineer trying out the sample. For example, "Make your data pipelines replayable and idempotent".
+1. **Key Concepts** - These are the DataOps and WAF concepts that the sample is showcasing. For example, "Continuous Integration". This will be the meat of your sample.
+1. **How to use the sample** - A clear deployment instructions and all pre-requisites along with a list of deployed resources.
+1. **Known Issues, Limitations and Workarounds** - Elaborate on any issues and deficiency in the provided solution the reader should be made aware of. Any key WAF concepts that the solution is **not** addressing that you particularly want to call out. Any workarounds in the deployment script.
+
+#### Automated Deployment
+
+> NOTE: This requirement is not yet consistently enforced but will eventually be required to facilitate automated testing of all deployment scripts of all samples.
+
+Ensure you have a `deploy.sh` or `deploy.ps1` in the root of the sample as the entry point of the deployment of your sample.
+
+## Contributing to an existing sample
+
+**First**, Look through Github issues if there are any similar work in progress (ie. issues with PRs associated to them) to avoid duplicate efforts.
+
+**Second**, Create a Github issue detailing your work using the "Feature Request" issue template. Tag the issue using the relevant label (ie. "single-tech: databricks", "e2e: fabric"). See "Labels" below.
+
+**Third**, Develop your changes. If you are not a member of the Azure-Samples organization (Microsoft employees only), you will need to fork the repository. Otherwise, familiarize yourself with the Branching structure (see below). Every sample should come with developer setup information and most come with a Devcontainer.
```

Comment:
```suggestion
**Third**, Develop your changes. If you are not a member of the [Azure-Samples](https://github.com/Azure-Samples) organization (Microsoft employees only), you will need to fork the repository. Otherwise, familiarize yourself with the [branching structure](#branching-structure). Every sample should include developer setup information, and most should provide a Devcontainer.
```